src/PHPMailer.php

```diff
@@ -818,6 +817,21 @@ public function addReplyTo($address, $name = '')
         return $this->addOrEnqueueAnAddress('Reply-To', $address, $name);
     }
 
+    private function enqueue($address, $data)
+    {
+        if (!array_key_exists($address, $data)) {
+            $data[$address] = $params;
+            return true;
+        }
+    }
+
+    private function throwException($error_message)
```

Comment:
This is a bad idea. Exceptions should be thrown from where they occur, otherwise you can't tell from the error alone where the error occurred and you have to generate a stack trace to find out.